Src/car.c

```diff
@@ -243,14 +243,15 @@ void taskBlink(void* can)
 		}
 		if (xSemaphoreTake(car.m_CAN, 100) == pdTRUE)
 		{
-			hcan1.pTxMsg = &tx;
-			HAL_CAN_Transmit(&hcan1, 100);
-			xSemaphoreGive(car.m_CAN);  //release CAN mutex
+//			hcan1.pTxMsg = &tx;
+//			HAL_CAN_Transmit(&hcan1, 100);
+//			xSemaphoreGive(car.m_CAN);  //release CAN mutex
+			xQueueSendToBack(car.q_txcan, &tx, 100);
 
 		}
 		//		//req regid 40
```

Comment:
The double comment // // yikes. I know @xbenng you said you were going to fix this, I just wanted to comment on something in this review to get practice reviewing since I haven't done it before.